sw/device/tests/power_virus_systemtest.c

```diff
@@ -526,37 +475,166 @@ static void configure_pinmux(void) {
   CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIor10,
                                         kTopEarlgreyPinmuxOutselPwmAonPwm5));
 
-  // SPI Host 1:
-  //    CSB on IOB0
-  //    SCK on IOB3
-  //    SD0 on IOA2
-  //    SD1 on IOR11
-  //    SD2 on IOR12
-  //    SD3 on IOR13
-  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob0,
+  // Apply this configuration only for the FPGA.
+  // For the simulation, apply the config in configure_pinmux_sim().
+  if (kDeviceType == kDeviceFpgaCw305 || kDeviceType == kDeviceFpgaCw310) {
+    // SPI Host 1:
+    //    CSB on IOB0
+    //    SCK on IOB3
+    //    SD0 on IOA2
+    //    SD1 on IOR11
+    //    SD2 on IOR12
+    //    SD3 on IOR13
+    CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob0,
+                                          kTopEarlgreyPinmuxOutselSpiHost1Csb));
+    CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob3,
+                                          kTopEarlgreyPinmuxOutselSpiHost1Sck));
+    CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa2,
+                                          kTopEarlgreyPinmuxOutselSpiHost1Sd0));
+    CHECK_DIF_OK(dif_pinmux_output_select(&pinmux,
+                                          kTopEarlgreyPinmuxMioOutIor11,
+                                          kTopEarlgreyPinmuxOutselSpiHost1Sd1));
+    CHECK_DIF_OK(dif_pinmux_output_select(&pinmux,
+                                          kTopEarlgreyPinmuxMioOutIor12,
+                                          kTopEarlgreyPinmuxOutselSpiHost1Sd2));
+    CHECK_DIF_OK(dif_pinmux_output_select(&pinmux,
+                                          kTopEarlgreyPinmuxMioOutIor13,
+                                          kTopEarlgreyPinmuxOutselSpiHost1Sd3));
+    CHECK_DIF_OK(dif_pinmux_input_select(
+        &pinmux, kTopEarlgreyPinmuxPeripheralInSpiHost1Sd0,
+        kTopEarlgreyPinmuxInselIoa2));
+    CHECK_DIF_OK(dif_pinmux_input_select(
+        &pinmux, kTopEarlgreyPinmuxPeripheralInSpiHost1Sd1,
+        kTopEarlgreyPinmuxInselIor11));
+    CHECK_DIF_OK(dif_pinmux_input_select(
+        &pinmux, kTopEarlgreyPinmuxPeripheralInSpiHost1Sd2,
+        kTopEarlgreyPinmuxInselIor12));
+    CHECK_DIF_OK(dif_pinmux_input_select(
+        &pinmux, kTopEarlgreyPinmuxPeripheralInSpiHost1Sd3,
+        kTopEarlgreyPinmuxInselIor13));
+  }
+}
+
+/**
+ * Configures pins for DVsim.
+ * In chip_if.sv, agents and interfaces are connected to fixed pins.
+ * To be able to use the agents (e.g, spi_device_agent1), the C code's pinmux
+ * settings must be compatible with the settings in chip_if.sv.
+ */
+static void configure_pinmux_sim(void) {
```

Comment:
To be compatible with chip_if.sv settings, in simulation, I need to change the pin configs for SPI HOST1, UART2, and PWM.

In the main function, this is how I configure the pinmux:

```
main(){
  pinmux_configure();
  if(simulation) {
    pinmux_configure_sim()
  }
}
```